rust-runtime/aws-smithy-runtime-api/src/client/identity.rs

```diff
@@ -285,14 +295,118 @@ impl Identity {
     pub fn expiration(&self) -> Option<SystemTime> {
         self.expiration
     }
+
+    /// Returns arbitrary data associated with this `Identity`.
+    pub fn get<T: Any + Debug + Send + Sync + 'static>(&self) -> Option<&T> {
```

Comment:
`get_property` or `get_attribute`? `get` is nice and short of course but my (very mild) concern or gut feeling is this makes this API seem like a core operation on identity (to me for some reason). Also aligns with builder `property` API defined below better perhaps.

Reply:
Renamed to `property` in 37c6d31 (since there is already a getter `.data` so didn't prefix it with `get_` for consistency).